etcdctl/README.md

```diff
@@ -160,9 +160,24 @@ The protobuf encoding of the DeleteRange [RPC response][etcdrpc].
 ```bash
 ./etcdctl put foo bar
 # OK
+./etcdctl put foo1 bar
```

Comment:
@sinsharat please split each example into a separate codeblocks so it's clear the examples are independent of each other?